opentelemetry-sdk/src/opentelemetry/sdk/resources/__init__.py

```diff
@@ -165,9 +166,27 @@ def create(
         """
         if not attributes:
             attributes = {}
-        resource = _DEFAULT_RESOURCE.merge(
-            OTELResourceDetector().detect()
-        ).merge(Resource(attributes, schema_url))
+
+        resource = _DEFAULT_RESOURCE
+
+        for resource_detector in environ.get(
+            OTEL_EXPERIMENTAL_RESOURCE_DETECTORS, "otel"
```

Comment:
Does this make sure to include `OTELResourceDetector` is the env is set to just `process`?

Reply:
It doesn't. I realize we used to always use an `OTELResourceDetector`, but should we do that? I can't find anywhere in the spec that requires us to always add an `OTELResourceDetector` and it may be an unexpected behavior for the user.

Reply:
It's not an unexpected behaviour. It is required by the spec. It adds the attributes from the env `OTEL_RESOURCE_ATTRIBUTES` and SDK is expected to pick up the env configuration all the time. If they don't configure the env it is an empty resource.

Reply:
Ok, added it.